script/check-posix-pty

```diff
@@ -155,9 +163,15 @@ EOF
 
     netstat -an | grep -q 5683 || die 'Application CoAP port is not available!'
 
-    extaddr=$(grep -azoP 'extaddr[\r\n]+\K[0-9a-z]{16}' $OT_OUTPUT | tr -d '\0')
+    extaddr=$(grep -ao -A +1 'extaddr' $OT_OUTPUT | tail -n1 | tr -d '\r\n\0')
     echo "Extended address is: ${extaddr}"
 
+    prefix=$(grep -ao 'Mesh Local Prefix: [0-9a-f:]\+' $OT_OUTPUT | cut -d: -f2- | tr -d ' \r\n')
+    LEADER_ALOC="${prefix}ff:fe00:fc00"
+    # Retrievie test resource through application CoAP
+
+    [[ $OSTYPE == "linux-gnu"* ]] || return 0
```

Comment:
why we don't do following tests in macOS ?

Reply:
see https://github.com/openthread/openthread/issues/6363

Reply:
Thanks

Reply:
Add comment with link to issue here?

Reply:
added